src/lib/getCrumb.ts

```diff
@@ -1,8 +1,5 @@
-import type { RequestInfo, RequestInit, Response } from "node-fetch";
 import type { ExtendedCookieJar } from "./cookieJar";
-// eslint-disable-next-line @typescript-eslint/ban-ts-comment
-// @ts-ignore: we have to ignore this for csm output.
-import pkg from "../../package.json" assert { type: "json" };
```

Comment:
I know you only added these recently @gadicc but looking at the commit log it seems like you added them to allow backwards support for Node 17 which is dropped by this PR, so, yay?

Reply:
Very off topic here, but this annoyingness was mostly related to Jest in this case. Curious on your thoughts on moving to [vitest](https://vitest.dev) for nicer ESM support, and also quicker tests

Reply:
> I know you only added these recently @gadicc but looking at the commit log it seems like you added them to allow backwards support for Node 17 which is dropped by this PR, so, yay?

Kinda.  This one was a total pain, because, newer nodes fail without the assertion, older nodes fail with it, and they didn't have a sufficient gap between enforcing - so became very tricky to support a range (also because it's at the static level and you can't do runtime checks).

The Node 17 issue was that it requires the older **`assert`** `{ type: "json" }` syntax vs the newer (and better supported?) **`with`** `{ type: "json" }` syntax that superseded it.  So it definitely helps that we're dropping Node < 18, but I think this is still required for newer versions.  Did you have any luck with a CLI run?  e.g. `yarn build` and then `yahoo-finance quote AAPL` from the command line?  This is where I ran into this.